backend/routers/memories.py

```
@@ -1,11 +1,12 @@
-from fastapi import APIRouter, Depends, HTTPException
+from fastapi import APIRouter, Depends, HTTPException, Body
 
 import database.memories as memories_db
 import database.redis_db as redis_db
 from database.vector_db import delete_vector
 from models.memory import *
 from routers.speech_profile import expand_speech_profile
 from routers.transcribe_v2 import retrieve_in_progress_memory
+from utils.llm import get_transcript_structure
```

Ruff

`utils.llm.get_transcript_structure` imported but unused

Remove unused import: `utils.llm.get_transcript_structure`

(F401)